lib/matplotlib/tests/test_compare_images.py

```diff
@@ -42,7 +42,8 @@
         # Now test the reverse comparison.
         ('all128.png', 'all127.png', 0, 1),
     ])
-def test_image_comparison_expect_rms(im1, im2, tol, expect_rms):
+def test_image_comparison_expect_rms(im1, im2, tol, expect_rms, tmp_path,
+                                                            monkeypatch):
```

Comment:
Should be aligned with opening parenthesis:
```suggestion
def test_image_comparison_expect_rms(im1, im2, tol, expect_rms, tmp_path,
                                     monkeypatch):
```